package.json

```diff
@@ -13,20 +13,20 @@
   },
   "dependencies": {
     "@material-ui/core": "^4.11.2",
-    "@plasmicapp/cli": "^0.1.70",
-    "@plasmicapp/plume": "^0.1.15",
+    "@plasmicapp/cli": "^0.1.78",
+    "@plasmicapp/plume": "^0.1.20",
     "@plasmicapp/react-web": "^0.2.14",
     "@testing-library/jest-dom": "^5.11.5",
-    "@testing-library/react": "^11.1.1",
-    "@testing-library/user-event": "^12.2.0",
-    "@types/faker": "^5.1.4",
-    "@types/jest": "^26.0.15",
-    "@types/node": "^12.19.3",
-    "@types/react": "^16.9.56",
-    "@types/react-dom": "^16.9.9",
-    "@types/recharts": "^1.8.16",
+    "@testing-library/react": "^11.2.2",
+    "@testing-library/user-event": "^12.6.0",
+    "@types/faker": "^5.1.5",
+    "@types/jest": "^26.0.19",
+    "@types/node": "^12.19.9",
+    "@types/react": "^16.14.2",
```

Comment:
@marcusellison can you check if this makes sense? We should try to stay up to date where we can. I would maybe also look into dependabot support for these kind of things if possible.